docs/spectator/lang/java/ext/jvm-classloading.md

```diff
@@ -0,0 +1,39 @@
+# ClassLoading
+
+Uses the [ClassLoadingMXBean](https://docs.oracle.com/javase/7/docs/api/java/lang/management/ClassLoadingMXBean.html)
```

Comment:
The javadoc link should get bumped to at least 8. Maybe 17 is a better option as it is the more recent LTS release.

https://docs.oracle.com/en/java/javase/17/docs/api/java.management/java/lang/management/ClassLoadingMXBean.html

Reply:
I did a pass through the whole set of docs and updated the obvious ones to version 17.

There were a few things like links to tutorials and Pre/PostConstruct that were not easy to remap.